96.md

```diff
@@ -143,9 +143,9 @@ The upload response is a json object as follows:
   // and, optionally, all file metadata the server wants to make available
   //
   // nip94_event field is absent if unsuccessful upload
-  nip94_event: {
+  "nip94_event2":{
```

Comment:
Is this correct?

Reply:
Based on online formatters it seems yes. But it won't affect how encoders interpret it. so I think we can go with the same way like all other JSONs in this repo.

Reply:
Fixed.